### PR TITLE
Cloud Build assets & system tests migration (AIP-47)

### PR DESCRIPTION
**Pull request by bkossakowska**
Cloud Build assets & system tests migration (AIP-47)

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/cloud/cloud_build/example_cloud_build.py [hunk 1]
```diff
@@ -42,88 +42,65 @@
...
+    "steps": [{"name": "ubuntu", "args": ['echo', 'Hello world']}],
 }
 # [END howto_operator_create_build_from_repo_body]
 
 
 with models.DAG(
-    "example_gcp_cloud_build",
-    start_date=START_DATE,
+    DAG_ID,
+    schedule_interval='@once',
```
Comment (josh-fell) [inline tests/system/providers/google/cloud/cloud_build/example_cloud_build.py]:
```suggestion
    schedule='@once',
```
Updating to the new `schedule` param. See #25410 and #25648.

File: tests/system/providers/google/cloud/cloud_build/example_cloud_build_trigger.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+create_build_from_repo_body: Dict[str, Any] = {
+    "source": {"repo_source": {"repo_name": GCP_SOURCE_REPOSITORY_NAME, "branch_name": "master"}},
+    "steps": [{"name": "ubuntu", "args": ['echo', 'Hello world']}],
+}
+# [END howto_operator_create_build_from_repo_body]
+
+
+with models.DAG(
+    DAG_ID,
+    schedule_interval='@once',
```
Comment (josh-fell) [inline tests/system/providers/google/cloud/cloud_build/example_cloud_build_trigger.py]:
```suggestion
    schedule='@once',
```
Same here. See #25410 and #25648.

**Review by josh-fell — APPROVED** [review]
LGTM. Only a couple small updates to use `schedule` in example DAGs now based on #25410 and #25648.

**Comment by josh-fell** [comment]
Well, approval with addressing the unit test failures too.

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Congrats @bkossakowska !

**Pull request merged**